microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,593 @@
+
+* **Pop-ups**: Pop-ups don't work well within WebView2 inside Unity apps on HoloLens 2, but they work fine in 2D XAML apps on the device. Avoid pop-ups and use alternative techniques or UI designs, such as custom pop-up-like elements within the WebView using HTML, CSS, and JavaScript.
+
+* **New windows**: WebView2 instances on HoloLens 2 navigate within the same window by default, unlike on Desktop. Follow this default behavior for a better user experience.
+
+* **Enterprise authentication**: Automatic Single Sign-On (SSO) leveraging OS-level tokens is currently not supported in WebView2 on HoloLens 2. Users can still sign in by providing credentials, except for cases requiring device-level authentication. Cookie storage works as expected.
+
+* **User interactions**: Unlike native HoloLens 2 slates, WebView2 is best interacted with using far-interaction hand rays. Touch-to-swipe and scroll interactions might not be supported.
+
+* **Performance**: Complex websites with heavy use of JavaScript or advanced rendering may impact system performance or the host application's framerate. For general performance-related limitations and recommendations, see [Understanding performance for mixed reality](/windows/mixed-reality/develop/advanced-concepts/understanding-performance-for-mixed-reality) in the Mixed Reality documentation. Also see [Performance optimization](#performance-optimization), below.
+
+
+<!-- ------------------------------ -->
+#### Navigation
+
+In step 6, we touched on some navigation methods. In this section, we'll expand on what we learned.
+
+**`IWebView` Interface**
+The `IWebView` interface exposes a few methods, events, and properties related to page navigation. The main functionality exposed here is the ability for developers to navigate to a given URL, by using `Load(Uri url)`:
+
+```C#
+public interface IWebView
+{
+    // Non-navigation methods are removed for clarity.
+
+    event WebView_OnNavigated Navigated;
+
+    Uri Page { get; }
+
+    void Load(Uri url);
+
+    void Reload(bool ignoreCache);
+}
+```
+
+**`IWithBrowserHistory` Interface**
+The `IWithBrowserHistory` interface exposes a few methods and events related to page navigation. This mainly allows developers to navigate forward and backward, as you would expect with a typical web-browsing experience:
+
+```C#
+public interface IWithBrowserHistory : IWebView
+{
+    // Non-navigation methods are removed for clarity.
+
+    event WebView_OnCanGoForwardUpdated CanGoForwardUpdated;
+
+    event WebView_OnCanGoBackUpdated CanGoBackUpdated;
+
+    void GoBack();
+
+    void GoForward();
+}
+```
+
+The `SetVirtualHostNameToFolderMapping` method in WebView2 enables mapping between a virtual host name and a folder path, making it accessible to websites using that host name. This method maps a local domain name to a local folder so that the WebView2 control loads content from the specified local folder when attempting to access a resource for that domain.
```

Comment:
SetVirtualHostNameToFolderMapping occurs 1x, seems to be wrong and means SetVirtualHostMapping so i'm locally changing to that  
@michaelfarnsworth

Reply:
done.  present Comment Outdated, see https://github.com/MicrosoftDocs/edge-developer/pull/2567/files
Find in .md view:
The `CoreWebView2.SetVirtualHostNameToFolderMapping` method enables mapping between a virtual host name and a folder path
Find in Rendered view:
The CoreWebView2.SetVirtualHostNameToFolderMapping method enables mapping between a virtual host name and a folder path

You can close thread

Reply:
changed to
```
The [CoreWebView2.SetVirtualHostNameToFolderMapping Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.setvirtualhostnametofoldermapping) enables mapping
```
you can close thread